fluentgen/src/test/java/com/azure/autorest/fluent/TestUtils.java

```diff
@@ -55,7 +55,7 @@ public static class MockFluentGen extends FluentGen {
             DEFAULT_SETTINGS.put("required-parameter-client-methods", true);
             DEFAULT_SETTINGS.put("generate-samples", true);
             DEFAULT_SETTINGS.put("generate-tests", true);
-            //DEFAULT_SETTINGS.put("client-flattened-annotation-target", "NONE");
+            DEFAULT_SETTINGS.put("client-flattened-annotation-target", "NONE");
```

Comment:
new record has client-flattened-annotation-target=NONE, so we can now enable it here as well.